documentation/oauth/README.md

```diff
@@ -0,0 +1,59 @@
+# Setting Up OAuth2.0 on Wordpress Using the WP OAuth Server Plug-in and Testing With Postman
+
+## Setting up the WP OAuth Server Plug-in
+- To set-up the use of OAuth2.0 on Wordpress, install the plug-in [WP OAuth Server](https://wordpress.org/plugins/oauth2-provider/), either through the Wordpress Dashboard or through the console into the `/var/www/html/wp-content/plugins/` directory.
+
+
+- Once WP OAuth Server plug-in has been installed, navigate to the plug-in by selecting the `OAuth Server` tab in the Wordpress Dashboard. Then, to add a new "Client," or application, select the `Clients` tab and press the `Add New Client` button.
+
+
+- Fill in the Client name (e.g. "Coveros Mobile App"), the Redirect URI (`https://www.getpostman.com/oauth2/callback` for this example testing with Postman), and a Client description (e.g. "An app for the Mobile Applications Testing course"). Press the `Create Client` button to continue.
+
+At this stage, you will have set up the Wordpress side of using OAuth2.0. For the Client you just created, you should see a "Client ID," as well as a "Client Secret," when you hover under the name of the Client and press the `Show Secret` link. You will use the "Client ID" and "Client Secret" in the next step--testing the OAuth2.0 connection with Postman.
+
+## Testing the OAuth2.0 Connection with Postman
+
+- Open Postman, or [install](https://www.getpostman.com/) the application if you have not already.
+
+
+- To start a test request, under the `Authorization` tab, which should already be selected, under the dropdown `Type` select `OAuth 2.0`. Press the `Get New Access Token` button that appears.
+
+
+- Fill out the Token Name (e.g. Coveros Dev Wordpress), the Auth URL (https://www3.dev.secureci.com/oauth/authorize), the Access Token URL (https://www3.dev.secureci.com/oauth/token), the Client ID and Client Secret (from above), and select `Authorization Code` as the Grant Type. Then press the `Request Token` button.
+
+This should bring up a  log-in window, in which you may enter your credentials and approve the OAuth2.0 connection. Note: only users with "Administrator" status will have credentials that will allow for a successful OAuth2.0 connection.
+
+- After logging in, given that valid credentials were used, Postman should now list the new token under "Existing Tokens." Select the token you just created, and then in the "Add token to" dropdown, select the "Header" option. Then, press the "Use Token" button.
+
+
+Now, let's make a request to test the OAuth2.0 connection. 
+
+- Select "POST" as the request type, then input `https://www3.dev.secureci.com/wp-json/wp/v2/comments?post=7509&content=<Some+comment+content>&author_name=A+fellow+cat+lover&author_email=catsrus@catlovers.com` as the URL. Then, press `Send`. 
```

Comment:
In the content descriptive tag you used plus signs but that might be a little confusing since its supposed to be a tag and not an actual value. If you use a tag, make it something more like `content=<SOME_COMMENT_CONTENT>` so it's really obvious